common/src/main/java/com/pg85/otg/generator/resource/AboveWaterGen.java

```diff
@@ -26,13 +28,19 @@ public void spawn(LocalWorld world, Random rand, boolean villageInChunk, int x,
         if (y == -1)
             return;
 
+       if (!bLoaded) {
```

Comment:
Could put this in a method in Resource instead, no repeating code for each Gen class?